### PR TITLE
Merge release 0.3.0

### PR DESCRIPTION
**Pull request by yunikkk**

### CODE REVIEW

File: Examples/src/main/res/values/developer-config.xml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="mapbox_access_token">pk.eyJ1IjoiZGVuaXNrb3JvbmNoaWstZGV2IiwiYSI6ImNqdHNqMTRsbjBwaWI0ZnFudTVzbWl6bXQifQ.hUcnwjXhZAMHO5leM8wSCw</string>
```
Comment (RingerJK) [inline Examples/src/main/res/values/developer-config.xml]: are you sure, that token should be in common access?

File: MapboxVision/src/main/java/com/mapbox/vision/location/android/AndroidLocationEngineImpl.kt [hunk 2]
```diff
@@ -8,71 +8,74 @@ import android.location.LocationManager
...
+            )
+        }
     }
 
     override fun onStatusChanged(provider: String?, status: Int, extras: Bundle?) {
-        Log.v(TAG, String.format("Provider %s status changed to %d (current provider is %s).",
-                provider, status, currentProvider))
+        VisionLogger.v(
+            TAG,
+            String.format(
```
Comment (RingerJK) [inline MapboxVision/src/main/java/com/mapbox/vision/location/android/AndroidLocationEngineImpl.kt]: kotlin style :) 
`"Provider $provider status changed to $status (current provider is $currentProvider)."
`
In 3 places

File: Examples/src/main/res/values/developer-config.xml (hunk 1 above)
Comment (yunikkk) [reply Examples/src/main/res/values/developer-config.xml]: Definitely no, I'll remove it and we'll rotate the token now, too.Nice catch!

File: MapboxVision/src/main/java/com/mapbox/vision/location/android/AndroidLocationEngineImpl.kt (hunk 2 above)
Comment (yunikkk) [reply MapboxVision/src/main/java/com/mapbox/vision/location/android/AndroidLocationEngineImpl.kt]: Lets do this in separate PR, it's just dev-master sync

Comment (RingerJK) [reply MapboxVision/src/main/java/com/mapbox/vision/location/android/AndroidLocationEngineImpl.kt]: sounds good

**Pull request merged**